fvp-psa-sp.mk

```diff
@@ -13,12 +13,15 @@ TS_UEFI_INTERNAL_CRYPTO	?= n
 SP_PACKAGING_METHOD		?= embedded
 SPMC_TESTS			?= n
 SPMC_AT_EL			?= 1
-TS_BTI_ENABLED			?= unset
```

Comment:
It doesn't make much sense to introduce `TS_BTI_ENABLED` in the parent commit only to remove it in this one. Please squash the two commits into "fvp: add PAC and BTI support".

Reply:
Yes, you are right. Squashed them into a single commit.